docs/csharp/how-to/index.md

```diff
@@ -0,0 +1,241 @@
+- [Determine if a string represents a number](../programming-guide/strings/how-to-determine-whether-a-string-represents-a-numeric-value.md).
+- [Use <xref:System.String.Split%2A> to separate strings](../programming-guide/strings/how-to-parse-strings-using-string-split.md).
+- [Combine multiple strings into one](../programming-guide/strings/how-to-concatenate-multiple-strings.md).
+- [Search for text in a string](../programming-guide/strings/how-to-search-strings-using-string-methods.md).
+- [Search strings using regular expressions](../programming-guide/strings/how-to-search-strings-using-regular-expressions.md).
+
+## Convert between types
+
+You may need to convert an object to a different type.
+
+- [Determine if a string represents a number](../programming-guide/strings/how-to-determine-whether-a-string-represents-a-numeric-value.md).
+- [Convert between strings that represent hexadecimal numbers and the number](../programming-guide/types/how-to-convert-between-hexadecimal-strings-and-numeric-types.md).
+- [Convert a string to a <xref:System.DateTime>](../programming-guide/strings/how-to-convert-a-string-to-a-datetime.md).
+- [Convert a byte array to an int](../programming-guide/types/how-to-convert-a-byte-array-to-an-int.md).
+- [Convert a string to a number](../programming-guide/types/how-to-convert-a-string-to-a-number.md).
+- [Use `as` and `is` to safely cast to a different type](../programming-guide/types/how-to-safely-cast-by-using-as-and-is-operators.md).
+- [Define conversion operators for `struct` types](../programming-guide/statements-expressions-operators/how-to-implement-user-defined-conversions-between-structs.md).
+- [Determine if a type is a nullable type](../programming-guide/nullable-types/how-to-identify-a-nullable-type.md).
+- [Convert between nullable and non-nullable value types](../programming-guide/nullable-types/how-to-safely-cast-from-bool-to-bool.md).
+
+## Equality and ordering comparisons
+
+You may create types that define their own rules for equality or define a
+natural ordering among objects of that type.
+
+- [Test for reference based equality](../programming-guide/statements-expressions-operators/how-to-test-for-reference-equality-identity.md).
+- [Define value based equality for a type](../programming-guide/statements-expressions-operators/how-to-define-value-equality-for-a-type.md).
+
+## Exception handling
+
+Throwing exceptions is how .NET programs report that methods did not
+successfully complete their work. In these articles you'll learn to work
+with exceptions.
+
+- [Handle exceptions using `try` and `catch`](../programming-guide/exceptions/how-to-handle-an-exception-using-try-catch.md).
+- [Cleanup resources using `finally` clauses](../programming-guide/exceptions/how-to-execute-cleanup-code-using-finally.md).
+- [Recover from non-CLS (Common Language Specification) exceptions](../programming-guide/exceptions/how-to-catch-a-non-cls-exception.md).
```

Comment:
This is a super-nit, but I don't think many people know what a non-CLS exception is, so I wonder if it's worthwhile to change the title. But on the other hand, if you don't know what it is, you probably don't need to handle it.

Reply:
Overall on this article, I took the 2nd approach: The How-To articles are meant to solve a specific problem, and the reader should be familiar with enough of the terms for their particular problem.